tl.tl

```diff
@@ -6509,17 +6517,16 @@ local function tl_package_loader(module_name: string): any
    if found_filename then
       local input = fd:read("*a")
       fd:close()
-      local tokens = tl.lex(input)
-      local errs = {}
-      local i, program = tl.parse_program(tokens, errs, found_filename)
-      local code = tl.pretty_print_ast(program, true)
-      local chunk = load(code, found_filename)
+      local chunk, err = tl.load(input, module_name, "t")
```

Comment:
I'm not sure by just looking at the diff, but: shouldn't it be found_filename here instead of module_name? So that any parse errors contain the correct filename

Reply:
Ah, yes it should.